<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MovieDB</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MovieDB</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Book</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Page</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Media</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>movie count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="movieCount">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addMovie">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add new movies...</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>add.png</normaloff>add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteMovie">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove the selected movie(s)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>remove.png</normaloff>remove.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort order:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sortOrder">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>by Title</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>by Book number</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="printButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Save printable copy of the movie list on disk</string>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>printer.png</normaloff>printer.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchTerm">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearSearch">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Delete the search text</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>delete.png</normaloff>delete.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
